packages/cli-next/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@novu/novu-labs",
-  "version": "0.24.2",
-  "description": "On-Boarding Cli",
+  "name": "novu-labs",
+  "version": "0.24.3-alpha.1",
+  "description": "Novu Labs CLI. Used to sign-up, sync with Novu Cloud, and run Novu Studio.",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** ‏ Can we actually include a deprecation warning here as well?

Reply:
good call! Updated